pages.pt_BR/linux/apt-get.md

```diff
@@ -23,6 +23,10 @@
 
 `apt-get upgrade`
 
+-  Limpar o repositório local — removendo os arquivos de pacotes (.deb) que não podem mais ser baixados (download interrompido) e são completamente inúteis e obsoletos.
```

Comment:
Perhaps this will read better?
```suggestion
-  Limpar o repositório local — removendo os arquivos de pacotes (.deb) de downloads interrompidos que não podem mais ser baixados.
```